examples/webhook.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from os import getenv
+
+from aiohttp.web import run_app
+from aiohttp.web_app import Application
+
+from aiogram.webhook.aiohttp_server import SimpleRequestHandler, setup_application
+from aiogram import Dispatcher, Router, Bot, types, F
+from aiogram.methods import SendMessage
+
+
+TELEGRAM_TOKEN = getenv("TELEGRAM_TOKEN")
+
+# Webhook settings
+WEBHOOK_HOST = getenv("WEBHOOK_HOST", "https://your.domain")
```

Comment:
The most frequently asked question about the webhook example in 2.x is the meaning of each property, so could you please explain that in the example?